Samples/Windowing/cs-winui/ZOrder.xaml.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.UI;
+using Microsoft.UI.Windowing;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Navigation;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using WinRT;
+
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace Windowing
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class ZOrder : Page
+    {
+        private AppWindow _mainAppWindow = MainWindow.AppWindow;
+        private int m_windowCount = 0;
+        List<WindowId> windowList = new List<WindowId>();
+
+        public ZOrder()
+        {
+            this.InitializeComponent();
+        }
+
+        private void SwitchZOrder(object sender, RoutedEventArgs e)
+        {
+            // Bail out if we don't have an AppWindow object.
+            if (_mainAppWindow != null)
+            {
+                switch (sender.As<Button>().Name)
+                {
+                    case "BottomZOrderBtn":
+                        _mainAppWindow.MoveInZOrderAtBottom();
+                        break;
+
+                    case "TopZOrderBtn":
+                        _mainAppWindow.MoveInZOrderAtTop();
+                        break;
+
+                    default:
+                        
+                        break;
+                }
+            }
+        }
+
+        private void myNewWindowButton_Click(object sender, RoutedEventArgs e)
+        {
+            // Count up the number of windows created so far, this is just used for Title generation so we don't collide
+            m_windowCount++;
+            Window nextWindow = new Window();
+            nextWindow.Content = new TextBlock() { Text = "A wild window appears!" };
```

Comment:
Maybe change the text that is displayed in the window here? :)
Something like "This window is intentionally left blank, it is only used to illustrate z-order management using the controls available in the app's main window."

Reply:
edited